canonical-kubernetes/addons/kubeflow/steps/01_install-ksonnet/after-deploy

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -eux
+
+. "$CONJURE_UP_SPELLSDIR/sdk/common.sh"
+
+export KUBECONFIG=$HOME/.kube/config.$JUJU_MODEL
+
+PATH="$PATH:$HOME/bin"
+
+if [[ $(uname -s) = "Darwin" ]]; then
+    platform="darwin"
+else
+    platform="linux"
+fi
+
+ksonnet_repo="https://github.com/ksonnet/ksonnet/releases/download/v$KS_VERSION"
+ksonnet_file="ks_${KS_VERSION}_${platform}_amd64.tar.gz"
+
+# always update the default config to the latest install
+echo "$HOME/.kube/config.$JUJU_MODEL" > "$HOME/.kube/config.conjure-up.default"
```

Comment:
this feels funny.  why is this step responsible for updating `$HOME/.kube/config.conjure-up.default`?

Reply:
It does feel funny to me as well, but it is what happens in helm as well..I (incorrectly?) assumed it was important for these 3rd party tools, but I don't see how it would be.

Reply:
Fixed this just for you, @kwmonroe